analysis/km_plot.py

```diff
@@ -0,0 +1,94 @@
+#################################################################################
+######################### figure 2 KM ###########################################
+#################################################################################
+import sys
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+from config import start_date, end_date, today
+sys.path.append('lib/')
+from functions import *
+
+#specify date columns
+date_cols = [
```

Comment:
nit: duplication of definition in counts.py, if its reused configuration maybe abstract to config.py?

Reply:
Thanks, moved to config.py.